config/systems.php

```diff
@@ -508,6 +508,41 @@
         'manufacturer' => '',
         'active' => false,
     ],
+    73 => [
+        'name' => 'Arcadia 2001',
+        'name_full' => 'Arcadia 2001',
+        'name_short' => 'Arcadia 2001',
+        'manufacturer' => '',
+        'active' => false,
+    ],
+    74 => [
+        'name' => 'Interton VC 4000',
+        'name_full' => 'Interton VC 4000',
+        'name_short' => 'Interton VC 4000',
+        'manufacturer' => '',
+        'active' => false,
+    ],
+    75 => [
+        'name' => 'Elektor TV Games Computer',
+        'name_full' => 'Elektor TV Games Computer',
+        'name_short' => 'Elektor TV Games Computer',
+        'manufacturer' => '',
+        'active' => false,
+    ],
+    76 => [
+        'name' => 'PC Engine CD',
+        'name_full' => 'PC Engine CD',
+        'name_short' => 'PC Engine CD',
+        'manufacturer' => '',
+        'active' => false,
+    ],
+    77 => [
+        'name' => 'Atari Jaguar CD',
+        'name_full' => 'Atari Jaguar CD',
+        'name_short' => 'Atari Jaguar CD',
+        'manufacturer' => 'Atari',
+        'active' => false,
+    ],
```

Comment:
Should probably toss 78 (Nintendo DSi) in here too.

'active' could be audited too. PS2, CPC, MegaDuck, Channel F, maybe others.

Reply:
Thanks! Addressed feedback; not sure about the other systems but I think it's good enough for now? It's mainly about not getting errors when systems are referenced in the releases file that are not yet seeded.